FreeAPS/Sources/Modules/Bolus/View/BolusRootView.swift

```diff
@@ -72,7 +72,7 @@ extension Bolus {
                                 autofocus: true,
                                 cleanInput: true
                             )
-                            Text("U").foregroundColor(.secondary)
+                            Text(state.amount > state.maxBolus ? "⚠️" : "U").foregroundColor(.secondary)
```

Comment:
See my other comment regarding `g`; same for the `U` abbrev. unit -->
`NSLocalizedString(" U", comment: "Insulin unit")`